boto/vendored/regions/exceptions.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
So the one thing that I am afraid of is if we update the vendored code and we forget to add these exceptions back, it could lead to runtime errors that are harder to find.

I really like this approach mainly because it is the easiest and straightforward to maintain as it requires the least amount of manual steps. Specifically there is no build step. All we have to do is copy and paste the file over. If we try to combine these two, there will be steps requiring to copy the exception into the regions.py file and remove the imports. So if we ever update this file again, I can see it be really easy to forget that step and I would not want this PR merged until there are tests in place to make sure we did the revendoring correctly, such as checking the exceptions.  Another bonus of keeping it this way is that if we do the copy and paste of the file over and we forget an exception or some other import, we will get import exceptions that will definitely cause the test to fail, without us needing specific tests for the vendored code.